views/js/qtiCreator/widgets/static/helpers/itemScrollingMethods.js

```diff
@@ -74,7 +74,7 @@ define(['jquery', 'util/typeCaster'], function ($, typeCaster) {
             const $form = widget.$form;
             let $wrapper =
                 wrapType === 'inner'
-                    ? widget.$container.find('[data-html-editable]').children(`.${wrapperTextCls}`)
+                    ? widget.$container.find('[data-html-editable]').first(`.${wrapperTextCls}`)
```

Comment:
```suggestion
                    ? widget.$container.children('[data-html-editable]').first(`.${wrapperTextCls}`)
```

Reply:
and also line 84 
find('[data-html-editable]') => .children('[data-html-editable]')